opacus/optimizers/optimizer.py

```diff
@@ -393,7 +399,7 @@ def clip_and_accumulate(self):
         """
 
         per_param_norms = [
-            g.view(len(g), -1).norm(2, dim=-1) for g in self.grad_samples
+            g.reshape(len(g), -1).norm(2, dim=-1) for g in self.grad_samples
```

Comment:
This has to do with some tensor memory allocation details specific to EW. The way I understand it, `view` can only work with contiguous inputs, which is not the case for EW grads

Reply:
I think we can skip the reshape and directly compute the norm as:
```python
g.norm(2, dim=tuple(range(1, g.ndim)))
```